### PR TITLE
Add 8-21 decisions for classification and part and fix rdau reference.

### PR DESCRIPTION
**Pull request by mlhale7**
## What does this Pull Request do?

This PR adds decisions for the top-level elements classification and part. It also changes how rdau:extent.en is referenced in the use case and justification descriptions so that it is consistent with the mapping.

## How should this be tested?

Check the syntax and content of the decisions.

## Interested parties

@markpbaggett @CanOfBees @mathewjordan

### CODE REVIEW

File: docs/source/contents/4_draft_mapping.rst [hunk 1]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
+This information is helpful to include as it provides information about where the physical item is shelved (though this
+is not a complete shelfLocator) and the broad subject the materials relate to.
+
+Xpath
+^^^^^
+
+mods:classification[@authority="lcc"] OR mods:classification
+
+Decision
+^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Decision
--------

```

File: docs/source/contents/4_draft_mapping.rst [hunk 2]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
+Some collections, like the University of Tennessee Commencements collection include full shelfLocators in the classification
+field (e.g. LD5297 .U55 2013). These should be edited before migration.
+
+Justification
+^^^^^^^^^^^^^
+This information is helpful to include as it provides information about where the physical item is shelved (though this
+is not a complete shelfLocator) and the broad subject the materials relate to.
+
+Xpath
+^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Xpath
-----

```

File: docs/source/contents/4_draft_mapping.rst [hunk 3]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
-==========
+Use case
+^^^^^^^^
+Some of our resources have already been formally cataloged and have a classification number. When these are available,
+they are included in the MODS metadata. Serials like the Alumnus and many of the Athletics media guides are good examples.
+Some collections, like the University of Tennessee Commencements collection include full shelfLocators in the classification
+field (e.g. LD5297 .U55 2013). These should be edited before migration.
+
+Justification
+^^^^^^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Justification
-------------

```

File: docs/source/contents/4_draft_mapping.rst [hunk 4]
```diff
@@ -690,15 +690,91 @@ typeOfResource
 classification
 ==============
 
-relateItem
-==========
+Use case
+^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: This is the wrong title level based on the context throughout the rest of the draft mapping doc.

This needs to be the following, or things won't build:

```
Use case
--------

```

File: docs/source/contents/4_draft_mapping.rst [hunk 5]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
 ========
 
 recordInfo
 ==========
 
+part
+====
+
+Use Case
+^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Use Case
--------

```

File: docs/source/contents/4_draft_mapping.rst [hunk 6]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
+
+Justification
+^^^^^^^^^^^^^
+
+Ultimately it was decided that this information is not important to keep because it is already present in the title field
+in both instances. With the Sanborn maps there is a difference between how the part is named - Sheet versus District-Ward,
+but it was not felt strongly that any additional remediation needed to be done.
+
+Xpath
+^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Xpath
-----

```

File: docs/source/contents/4_draft_mapping.rst [hunk 7]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
+====
+
+Use Case
+^^^^^^^^
+
+The MODS part element is infrequently used to describe a portion of a larger resource. In UTK's metadata, <part> is used
+in two collections - Great Smoky Mountains Colloquy and Sanborn Fire Insurance Map Collection.
+
+Justification
+^^^^^^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Justification
-------------

```

File: docs/source/contents/4_draft_mapping.rst [hunk 8]
```diff
@@ -690,15 +690,91 @@ typeOfResource
...
+in both instances. With the Sanborn maps there is a difference between how the part is named - Sheet versus District-Ward,
+but it was not felt strongly that any additional remediation needed to be done.
+
+Xpath
+^^^^^
+
+mods:part
+
+Decision
+^^^^^^^^
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Same as above:

```
Decision
--------

```

**Review by markpbaggett — CHANGES_REQUESTED** [review]
@mlhale7 there are a lot of title level problems that are going to stop the document from generating properly. Here's one way to get these errors to go away.

While these are just warnings, if we don't correct, it will make future pull requests nearly impossible to review.

**Comment by mlhale7** [comment]
@markpbaggett - let me know if any issues still remain. Apologies for not paying enough attention to notice that classification and part don't go as deep as the other top level elements.

**Comment by mlhale7** [comment]
Closing and reopening as I can't seem to fix the merge conflict.

**Pull request closed without merging**